web/src/features/exchanges/ExchangeArrow.tsx

```diff
@@ -16,7 +15,7 @@ interface ExchangeArrowProps {
   data: ExchangeArrowData;
   viewportWidth: number;
   viewportHeight: number;
-  map: MapboxMap;
+  map: maplibregl.Map;
```

Comment:
How does this actually work when the import is not defined and we don't set it as a global value anywhere? 🤔

Reply:
Maplibre is now exporting its own types that TS is picking up and since it's using a namespace it don't need importing. Importing Map directly also conflicts with the built-in Map type so I thought this was better.

Reply:
Ah okay, fair enough - it's a bit too much "magic" in my opinion when you can use something that isn't declared anywhere, but if it works and it's only for types I guess it can't hurt :D

Reply:
It's the same as the react and html or the builtin types that "just work".